src/Propel/Generator/Schema/Dumper/XmlDumper.php

```diff
@@ -503,8 +503,10 @@ private function appendIndexNode(Index $index, \DOMNode $parentNode)
     /**
      * Appends the generated <unique> XML node to its parent node.
      *
-     * @param Unique   $unique     The Unique model instance
+     * @param Unique   $index
```

Comment:
All object types should be FQCN, but this could be outside of this PR scope, maybe a follow up.